modules/win-instance/watchmaker-win-instance.template.cfn.yaml

```diff
@@ -631,19 +638,20 @@ Resources:
         watchmaker-launch:
           commands:
             10-watchmaker-launch:
-              command: !Sub 
+              command: !Sub
                 - >-
                   ${local_ShellCommandMap} -Command "Invoke-Command -ScriptBlock {
                   $ErrorActionPreference = 'Stop';
-                  c:\cfn\scripts\update-path.ps1 -ErrorAction Stop  -Verbose; 
+                  c:\cfn\scripts\update-path.ps1 -ErrorAction Stop  -Verbose;
                   watchmaker --log-level debug --log-dir C:\Watchmaker\Logs --no-reboot
                   ${local_UseWamConfig}
                   ${local_UseEnvironment}
                   ${local_UseOuPath}
                   ${local_UseComputerName}
                   ${local_UseAdminGroups}
-                  ${local_UseAdminUsers};
-                  if (-not $?) { throw 'watchmaker execution failed!' } 
+                  ${local_UseAdminUsers}
+                  !Ref ${WatchmakerExtraArgs};
```

Comment:
because this is a `Fn::Sub` block you can write this as just `${WatchmakerExtraArgs}`

Reply:
Oops, that's right, we're already in a `!Sub`!!!

Reply:
Want me to make the same change on the Linux templates?

Reply:
yes please